### DISCUSSION TITLE
Using Nuxt as Qwik

### DISCUSSION THREAD

**Discussion by othy54** [nuxt/nuxt#18522] [Questions]
Hi there !

I'm wondering if there is a way that I can use Nuxt 3 as Qwik or Astro, I mean generate a fully static website wich render only HTML and hydate JS only when need, like you see the button which increment a variable but it's only the html and when you click on this button it hydrate the JS needed to this button ?

Thanks

**Comment by TheAlexLichter** [comment]
Keep an eye on the space! There is lots of traction there right now. See e.g.
https://github.com/nuxt/framework/pull/5689

**Reply by othy54** [reply]
So there is a way right now with the experimental islands component to load a entire static website with lazy hydration ?

**Comment by tasiotas** [comment]
how is server component different from lazy component ?
